synapse/tests/test_lib_storm.py

```diff
@@ -3598,16 +3598,27 @@ async def test_storm_help_cmd(self):
             self.stormIsInPrint('testcmd', msgs)
             self.stormIsInPrint(': test command', msgs)
 
-            msgs = await alist(core.storm('help testcmd'))
+            msgs = await core.stormlist('help testcmd')
             self.stormIsInPrint('testcmd', msgs)
             self.stormNotInPrint('view.merge', msgs)
 
-            msgs = await alist(core.storm('[test:str=uniq] | help $node.value'))
+            msgs = await core.stormlist('[test:str=uniq] | help $node.value')
             self.stormIsInPrint('Get the value of the primary property of the Node.', msgs)
-            msgs = await alist(core.storm('[test:str=uniq] | help $node.value()'))
+
+            msgs = await core.stormlist('[test:str=uniq] | help $node.value()')
             self.stormNotInPrint('get the value of the primary property of the Node.', msgs)
```

Comment:
I know you didn't change this but I notice a similar line with a capital G a few lines above and I'm wondering if this assertion is passing because of a mismatch in case?

Reply:
This test shows the difference between the help value being a bound method `$node.value` and the resolved value `$node.value()` which is the string `uniq` which is then matched against the list of commands and types.

Reply:
Fixed the typo though!